resolwe/flow/executors/docker/run.py

```diff
@@ -153,9 +184,9 @@ def start(self):
 
     def run_script(self, script):
         """Execute the script and save results."""
-        mappings = SETTINGS.get('FLOW_DOCKER_MAPPINGS', {})
-        for map_ in mappings:
-            script = script.replace(map_['src'], map_['dest'])
+        # XXX: This is a huge hack and should be removed.
```

Comment:
Can we mount everything on the same path as it is outside?

Reply:
I don't think this is the correct way to go as it is much cleaner to have fixed volumes inside the container, regardless of what is outside. In this way you can always rely on specific paths when you run inside Docker.

It would be much better to always defer path translation to the executor (actually to the prepare part of the executor) as it is executor specific (e.g., the local executor does nothing, the Docker executor uses fixed volumes, etc.). Not just blindly using `*_DIR` from settings.

Reply:
This hack has been removed in the latest commit.